plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -3927,6 +3927,40 @@ private void assertFilterPushdown(
         });
     }
 
+    @Test
+    public void testCreateExternalTableWithInaccessibleSchemaLocation()
```

Comment:
i know that you copied name from existing test but I'd suggest to change it, inaccessible suggests that there is a permission issue or something like that when test actually checks scenario in which location is nonexisting.